Packages/sitepackage/Configuration/TypoScript/setup.typoscript

```diff
@@ -0,0 +1,30 @@
+######################
+#### DEPENDENCIES ####
+######################
+<INCLUDE_TYPOSCRIPT: source="FILE:EXT:bootstrap_package/Configuration/TypoScript/setup.typoscript">
```

Comment:
let's use the new `@` syntax

Reply:
Yeah sure. That's from the Sitepackagebuilder, have to change it there too :)